bin/satellite-clone

```diff
@@ -23,10 +23,6 @@ def yes_or_no(question):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--start-at-task",
```

Comment:
What did these do? I am guessing they were for 6.4 and below support.

Reply:
this was an ansible argument that could be passed along, so you could `satellite-clone --start-at-task foo` and it would call `ansible-playbook --start-at-task foo satellite-clone-playbook.yml`. It let you start at that specific task.

But this involved trying to keep the state the same, and caching of facts, which in practice never worked well. I just removed the whole feature and the fact caching here.

Reply:
+1 to removing, not much used imo